climada/util/alpha_shape.py

```diff
@@ -0,0 +1,99 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Compute the concave hull of a set of points.

```

Comment:
It it no very clear to me why this is  an entire util submodule and why it was named `alpha_shape.py`. Could you please explain?

Reply:
We can discuss if this should be part of the pull request.

This util function is used to calculate the burned area of a fire. At the time it's not used for anything but it could become more important as BA is something which is often communicated as an impact. Thus, it's nice that we produce that number but it doesn't necessarily be part of this pull request

Reply:
I've removed the alpha_shape and the hull_bruned_area method in commit 13a2c8cb as calculating area burned is not used and the module is complex already

Reply:
Fair enough. It would be useful if you maybe could make a branch containing simply this module for potential future use.